<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recite</class>
 <widget class="QDialog" name="recite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>368</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>459</width>
    <height>302</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>42</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>42</width>
     <height>22</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(196, 47, 47);</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>181</width>
     <height>41</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>181</width>
     <height>41</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>151</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>151</width>
     <height>16</height>
    </size>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:7pt; font-weight:600; color:#ffffff;&quot;&gt;请输入上述单词释义&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>371</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>371</width>
     <height>291</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(105, 146, 200);
background-color: rgb(84, 113, 157);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>16</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:7pt; font-weight:600;&quot;&gt;最低错误次数&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Adobe 黑体 Std R</family>
      <pointsize>7</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 75 italic 9pt &quot;Adobe Arabic&quot;;
font: 12pt &quot;Adobe Arabic&quot;;

color: rgb(59, 80, 112);
font: 7pt &quot;Adobe 黑体 Std R&quot;;</string>
    </property>
    <property name="text">
     <string>开始！</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>131</width>
      <height>261</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>131</width>
      <height>261</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>131</width>
      <height>261</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(96, 192, 221);
background-color: rgb(121, 171, 185);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>81</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>81</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#ffffff;&quot;&gt;已经过词&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>100</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#ffffff;&quot;&gt;还剩词汇&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>64</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>64</width>
       <height>23</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>64</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>64</width>
       <height>23</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>80</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>15</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 6pt &quot;Adobe 黑体 Std R&quot;;
color: rgb(157, 116, 153);</string>
     </property>
     <property name="text">
      <string>返回上层菜单</string>
     </property>
    </widget>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
    <zorder>lcdNumber</zorder>
    <zorder>lcdNumber_2</zorder>
    <zorder>pushButton_3</zorder>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>291</width>
     <height>101</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>291</width>
     <height>101</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>291</width>
     <height>101</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(196, 125, 125);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>16</height>
     </size>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; font-weight:600; color:#ffffff;&quot;&gt;回答结果&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>91</width>
      <height>31</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>31</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Adobe 黑体 Std R&quot;;
background-color: rgb(255, 255, 255);
color: rgb(196, 51, 51);</string>
    </property>
    <property name="text">
     <string>就是这个！</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>201</width>
     <height>91</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>201</width>
     <height>91</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>201</width>
     <height>91</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>171</width>
     <height>81</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>171</width>
     <height>81</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/4db4fc3edf911c4ae8c1599931083c2.jpg);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame_2</zorder>
  <zorder>frame_4</zorder>
  <zorder>spinBox</zorder>
  <zorder>textBrowser</zorder>
  <zorder>label_2</zorder>
  <zorder>textBrowser_2</zorder>
  <zorder>frame_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
